packages/next/client/components/react-dev-overlay/internal/helpers/getRawSourceMap.ts

```diff
@@ -1,10 +1,10 @@
 import dataUriToBuffer, {
   MimeBuffer,
 } from 'next/dist/compiled/data-uri-to-buffer'
-import type { RawSourceMap } from 'source-map'
+import type { EncodedSourceMap } from '@jridgewell/source-map'
```

Comment:
```suggestion
import type { EncodedSourceMap } from 'next/dist/compiled/@jridgewell/source-map'
```

The type should be available from `next/dist/compiled` as well (since its type definition has been declared in `misc.d.ts`).

Reply:
Done. And I fixed `packages/next/build/webpack/loaders/resolve-url-loader/index.js`, too.